pymc4/distributions/distribution.py

```diff
@@ -4,6 +4,7 @@
 
 from tensorflow_probability import distributions as tfd
 from pymc4.coroutine_model import Model, unpack
+from pymc4.distributions.plate import Plate
```

Comment:
these should be relative imports

Reply:
Relative imports were discussed in #139, and @ferrine commented that he adopted absolute imports to get doctests to work correctly. I actually would prefer to use relative imports, but I haven't actually looked at the problems that doctests have when the imports are relative, and how they could be fixed.